ibis/backends/tests/test_window.py

```diff
@@ -103,8 +103,13 @@ def calc_zscore(s):
             ),
             id="nth",
             marks=[
-                pytest.mark.notimpl(["bigquery", "pandas", "snowflake"]),
+                pytest.mark.notimpl(["pandas", "snowflake"]),
                 pytest.mark.notyet(["impala", "mssql"]),
+                pytest.mark.broken(
+                    "bigquery",
+                    strict=False,
+                    reason="output types is float64 instead of the expected float32",
```

Comment:
Interesting, it sounds like there's maybe some missing type conversion code in BigQuery's `.execute` method.

Reply:
In any case, that's a good follow up.